keras/ops/nn.py

```diff
@@ -508,9 +509,9 @@ def softmax(x, axis=-1):
             else:
                 new_shape.append(original_shape[i])
                 i += 1
-        x = x.reshape(new_shape)
+        x = numpy.reshape(x, new_shape)
```

Comment:
We don't normally use `ops` in Keras code, prefer using `backend.reshape`.

`ops` are mostly either user-facing, or used when the input might be a KerasTensor.

Reply:
Switched it to backend - thank you!